packages/browser/src/eventbuilder.ts

```diff
@@ -162,7 +162,7 @@ export function eventFromException(
   const syntheticException = (hint && hint.syntheticException) || undefined;
   const event = eventFromUnknownInput(exception, syntheticException, attachStacktrace);
   addExceptionMechanism(event); // defaults to { type: 'generic', handled: true }
-  event.level = Severity.Error;
+  event.level = 'error' as Severity;
```

Comment:
Can we leave a comment near the `Severity` enum defn about why we have to do this cast everywhere?

Reply:
I'm wondering now if we shouldn't actually just change the places where `Severity` is used to accept either a `Severity` or a `SeverityLevel`...